bin/eslint_d.js

```diff
@@ -31,7 +31,16 @@ if (cmd === 'start') {
   } else {
     var commands = ['stop', 'status', 'restart'];
     if (commands.indexOf(cmd) === -1) {
-      client.lint(process.argv.slice(2));
+      var concat = require('concat-stream');
```

Comment:
The background is this: The only reason why `eslint_d` is any faster than `eslint` is because it's loading all the dependencies in the server process. This code here is executed on the client process which should be as lightweight as possible. Therefore, I might try to remove the dependency in the future again, but for now it's better to have this feature implemented than squeezing the last 5 milliseconds out of it :wink: